chainer_chemistry/datasets/citation_network/cora/.gitignore

```diff
@@ -0,0 +1,3 @@
+cora.cites
```

Comment:
remove this file. I think dataset should be inside `examples` directory, not inside library.

Reply:
fixed!